### PR TITLE
Debugging Rector run :)

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: services/Intranet/Autoloader.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Autoloader {
      * Autoloader
      * @param $className
      */
-    protected static function autoload($className) {
+    public static function autoload($className) {
```
Comment (TomasVotruba) [inline services/Intranet/Autoloader.php:12]: Here was the problem

File: .github/workflows/rector_run.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            # see https://github.com/shivammathur/setup-php
+            -
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none
+
+            -   uses: "ramsey/composer-install@v1"
+
+            -   run: vendor/bin/rector --debug
```
Comment (TomasVotruba) [inline .github/workflows/rector_run.yaml:23]: Could you enable the workflow run for me? This would give us debug output quickly

Comment (kjeanson) [reply .github/workflows/rector_run.yaml:23]: Having changed `protected` to `public` property on my autoloader

Here is my workflow

```
C:\wamp\www\RectorPHPExcel>vendor\bin\rector process tests --debug
[parsing] tests/Services/PHPExcelTest.php
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 526
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 527
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 538
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 539
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 541
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\String.php o
n line 542
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
2
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
3
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
6
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
7
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
7
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 81
9
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 82
0
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 82
0
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 82
0
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell.php on line 82
4
[refactoring] tests/Services/PHPExcelTest.php
    [applying] Rector\Renaming\Rector\Name\RenameClassRector
    [applying] Rector\Renaming\Rector\MethodCall\RenameMethodRector
    [applying] Rector\PHPOffice\Rector\StaticCall\AddRemovedDefaultValuesRector
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2551
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2551
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2672
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2672
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2676
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2764
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2768
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 2780
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3034
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3166

line 3168
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3169
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3457
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3457
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3460
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3461
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3891
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3891
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3936
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3942
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 3998
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Calculation.php on
line 4001
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell\DefaultValueBi
nder.php on line 82
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell\DefaultValueBi
nder.php on line 90
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Cell\DefaultValueBi
nder.php on line 90
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel2003XML
.php on line 692
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel2003XML
.php on line 701
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 1928
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 1991
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2128
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2153
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2163
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2181
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2194
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2198
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2278
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2402
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2402
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2402
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2417
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2417
o
n line 2417
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2432
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2432
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2432
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2447
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2447
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2447
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2462
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2462
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2462
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2477
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2477
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2477
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2492
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2492
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2492
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2507
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2507
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2507
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2549
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2614
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2630
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2808
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2891
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2958
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 2980
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3886
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3909
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3909
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3909
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3921
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3922
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3923
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3926
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3927
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3928
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3929
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3932
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3933
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3934
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3935
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 3999
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 4065
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 4068
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 4450
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 4601
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 4603
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 6889
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 6892
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 7297
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 7300
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\Excel5.php o
n line 7303
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 164
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 252
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 266
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 296
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 330
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 339
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 369
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 387
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Reader\SYLK.php on
line 436
Warning: "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shar
ed\OLE.php on line 288
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Shared\OLE.php on l
ine 446
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Worksheet\AutoFilte
r.php on line 720
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Worksheet\AutoFilte
r.php on line 720
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Writer\Excel5\Workb
ook.php on line 667
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Writer\Excel5\Workb
ook.php on line 667
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Writer\Excel5\Workb
ook.php on line 667
Deprecated: Array and string offset access syntax with curly braces is deprecated in C:\wamp\www\RectorPHPExcel\vendor\phpoffice\phpexcel\Classes\PHPExcel\Writer\Excel5\Works
heet.php on line 879
[post rectors] tests/Services/PHPExcelTest.php
[printing skipped due error] tests/Services/PHPExcelTest.php


                                                                                                                        
 [OK] Rector is done!
```

Comment (TomasVotruba) [reply .github/workflows/rector_run.yaml:23]: I mean the Github Actions, so we can see passing/broken CI here

File: rector.php [hunk 3]
```diff
@@ -3,11 +3,23 @@
...
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $parameters = $containerConfigurator->parameters();
     $parameters->set(Option::PATHS, [__DIR__ . '/tests']);
-    $parameters->set(Option::SETS, [SetList::PHPEXCEL_TO_PHPSPREADSHEET]);
+    $services = $containerConfigurator->services();
+
+    // this rule is causing troubles
+    $services->set(AddRemovedDefaultValuesRector::class);
```
Comment (TomasVotruba) [inline rector.php]: This rule is somehow crypling the whole class rename

**Comment by TomasVotruba** [comment]
So it seems the Excel classes are included somewhere...

This one fails with exception on autoload. If you remove the exception check, it 

image


It's also in DomPDF.php and MPdf.php classes. If all 3 `throw new ...` are commented out, it passes :+1: 

I'll see if it works with excel as dependency, because it breaks lot of things.

**Comment by TomasVotruba** [comment]
Peek 2021-04-28 16-22

**Comment by kjeanson** [comment]
@TomasVotruba A very big thanks for your help and your quick response! 👍 👍 👍

**Comment by TomasVotruba** [comment]
You're welcome 🙂

**Pull request merged**